tests/conftest.py

```diff
@@ -164,20 +156,42 @@ def fixture_analysis_family(case_id: str, family_name: str, ticket_nr: int) -> d
 # Config fixtures
 
 
+@pytest.fixture(name="base_config_dict")
```

Comment:
cant this just reuse the fixture of context_config?

Reply:
I like to have a small one that you can fill with whatever you like in the contests